tests/app/views/contexts/test_section_summary_context.py

```diff
@@ -156,6 +156,7 @@ def test_context_for_section_list_summary(people_answer_store):
         "summary": {
             "answers_are_editable": True,
             "collapsible": False,
+            "sections": {"groups": None},
```

Comment:
Why would this happen?. Add this if we have one?

Reply:
This happens at the moment when we have a section with just a list collector and a custom summary.

Reply:
As above. Add this if we have sections.

Reply:
Updated 👍